### ISSUE TITLE
Build failed while updating carthage

### ISSUE THREAD

**Issue by w00ker** [zvonicek/ImageSlideshow#253]
*** Building scheme "ImageSlideshow_framework" in ImageSlideshow.xcworkspace
 Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/.....
This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/z7/n4ny4l_n15lbhcgrysf35kk40000gn/T/carthage-xcodebuild.cRfy4y.log

**Comment by Thiryn** [comment]
yep, got the same
```
ImageSlideshow/Classes/Core/ImageSlideshow.swift:66:37: error: use of undeclared type 'PageIndicatorPosition'
    open var pageIndicatorPosition: PageIndicatorPosition = PageIndicatorPosition() {
                                    ^~~~~~~~~~~~~~~~~~~~~
ImageSlideshow/Classes/Core/ImageSlideshow.swift:66:14: error: could not infer type for 'pageIndicatorPosition'
    open var pageIndicatorPosition: PageIndicatorPosition = PageIndicatorPosition() {
             ^

** ARCHIVE FAILED **
```

**Comment by PH9** [comment]
Same here

> *** Building scheme "ImageSlideshow_framework" in ImageSlideshow.xcworkspace
> Build Failed
> 	Task failed with exit code 65:
> 	/usr/bin/xcrun xcodebuild -workspace /Users/username/iOSRTR/Carthage/Checkouts/ImageSlideshow/Example/ImageSlideshow.xcworkspace -scheme ImageSlideshow_framework -configuration Release -derivedDataPath /Users/username/Library/Caches/org.carthage.CarthageKit/DerivedData/9.4_9F1027a/ImageSlideshow/1.6.0 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/jl/cwd7f_tn0z10hgzqfmv8xt5r0000gn/T/ImageSlideshow SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/username/iOSRTR/Carthage/Checkouts/ImageSlideshow)
> 
> This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/jl/cwd7f_tn0z10hgzqfmv8xt5r0000gn/T/carthage-xcodebuild.z8odoG.log

**Comment by robjperez** [comment]
I submitted a PR with a patch to fix the build process:

https://github.com/zvonicek/ImageSlideshow/pull/258

**Comment by zvonicek** [comment]
Fixed in #258.
